File: app/src/main/java/com/goldmines/pojo/AnswerItem.java
package com.goldmines.pojo;

public class AnswerItem {
    private Explain explain;

    private String wasRendered;

    private String color;

    private String correct;

    private String signature;

    private String hints;

    private String align;

    private String essayid;

    private String content;

    private String size;

    private String location;

    private String locked;

    private String scrollable;

    private String font;

    public Explain getExplain ()
    {
        return explain;
    }

    public void setExplain (Explain explain)
    {
        this.explain = explain;
    }

    public String getWasRendered ()
    {
        return wasRendered;
    }

    public void setWasRendered (String wasRendered)
    {
        this.wasRendered = wasRendered;
    }

    public String getColor ()
    {
        return color;
    }

    public void setColor (String color)
    {
        this.color = color;
    }

    public String getCorrect ()
    {
        return correct;
    }

    public void setCorrect (String correct)
    {
        this.correct = correct;
    }

    public String getSignature ()
    {
        return signature;
    }

    public void setSignature (String signature)
    {
        this.signature = signature;
    }

    public String getHints ()
    {
        return hints;
    }

    public void setHints (String hints)
    {
        this.hints = hints;
    }

    public String getAlign ()
    {
        return align;
    }

    public void setAlign (String align)
    {
        this.align = align;
    }

    public String getEssayid ()
    {
        return essayid;
    }

    public void setEssayid (String essayid)
    {
        this.essayid = essayid;
    }

    public String getContent ()
    {
        return content;
    }

    public void setContent (String content)
    {
        this.content = content;
    }

    public String getSize ()
    {
        return size;
    }

    public void setSize (String size)
    {
        this.size = size;
    }

    public String getLocation ()
    {
        return location;
    }

    public void setLocation (String location)
    {
        this.location = location;
    }

    public String getLocked ()
    {
        return locked;
    }

    public void setLocked (String locked)
    {
        this.locked = locked;
    }

    public String getScrollable ()
    {
        return scrollable;
    }

    public void setScrollable (String scrollable)
    {
        this.scrollable = scrollable;
    }

    public String getFont ()
    {
        return font;
    }

    public void setFont (String font)
    {
        this.font = font;
    }

    @Override
    public String toString()
    {
        return "AnswerItem [explain = "+explain+", wasRendered = "+wasRendered+", color = "+color+", correct = "+correct+", signature = "+signature+", hints = "+hints+", align = "+align+", essayid = "+essayid+", content = "+content+", size = "+size+", location = "+location+", locked = "+locked+", scrollable = "+scrollable+", font = "+font+"]";
    }
}

File: app/src/main/java/com/goldmines/jsonkotlinpojos/AnswerItem.kt
package com.goldmines.jsonkotlinpojos


import com.google.gson.annotations.SerializedName

data class AnswerItem(
        @SerializedName("align")
        val align: String, // 0
        @SerializedName("color")
        val color: String, // 0,0,0
        @SerializedName("content")
        val content: String, // <B><I>x</B></I> x 2
        @SerializedName("correct")
        val correct: String, // false
        @SerializedName("essayid")
        val essayid: String, // 0
        @SerializedName("explain")
        val explain: Explain,
        @SerializedName("font")
        val font: String, // Arial
        @SerializedName("hints")
        val hints: String, // False, False, False, False
        @SerializedName("location")
        val location: String, // 160,361,97,27
        @SerializedName("locked")
        val locked: String, // false
        @SerializedName("scrollable")
        val scrollable: String, // false
        @SerializedName("signature")
        val signature: String, // AnswerR
        @SerializedName("size")
        val size: String, // 14
        @SerializedName("wasRendered")
        val wasRendered: String // false
)